version-bump-strategy.md

```diff
@@ -0,0 +1,45 @@
+# Package version bump strategy
+
+## What do we have
+1) All guides (except IIS 10 [^1]) instructs to do `composer install --no-dev`. So user will install packages locked at `composer.lock`. 
+2) That means we control what package versions will be installed. We do it via `composer update` of `composer.lock`. 
+3) Internal technology wiki already mentions [^2] that PHP version requirement **must not** be ignored. 
```

Comment:
@alexanderko Let's remove the internal technology wiki reference as this guide will be published to customers. Thank you!